### ISSUE TITLE
Blue tile obscures home page header text

### ISSUE THREAD

**Issue by timothyfcook** [saxifrage/learn#313]
![screen shot 2015-10-30 at 9 35 05 am](https://cloud.githubusercontent.com/assets/1823300/10846970/b77a727a-7ee9-11e5-9fa5-76c32778fe80.png)

**Comment by timothyfcook** [comment]
Not enough contrast. Maybe we can dull the brightness on the highlighted tiles and change the font from blue to Black?

**Comment by cameronscott137** [comment]
Yes, will change headline from blue to black, and will drop the opacity on
each tile so that it's a bit less overwhelming.

On Fri, Oct 30, 2015 at 9:37 AM, Tim Cook notifications@github.com wrote:

> Not enough contrast. Maybe we can dull the brightness on the highlighted
> tiles and change the font from blue to Black?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/saxifrage/cityasacampus/issues/313#issuecomment-152527994
> .

## 

Cameron
(610) 883-7342
